jwst/ramp_fitting/tests/test_ramp_fit_step.py

```diff
@@ -0,0 +1,256 @@
+import pytest
+import numpy as np
+
+from jwst.ramp_fitting.ramp_fit_step import RampFitStep
+
+from jwst.datamodels import RampModel
+from jwst.datamodels import GainModel, ReadnoiseModel
+
+
+@pytest.fixture(scope="module")
+def generate_miri_reffiles(tmpdir_factory):
+    gainfile = str(tmpdir_factory.mktemp("data").join("gain.fits"))
+    readnoisefile = str(tmpdir_factory.mktemp("data").join('readnoise.fits'))
+
```

Comment:
Is there a reason to have `tmpdir_factory` here and have it scoped to module?  This fixture doesn't write anything out.  The mock reffiles are just returned in memory.

Reply:
The noise models are getting written to the temp directory.  Only the file names are being returned, not the reference to the models in memory.